### ISSUE TITLE
Air 33.1.1.217 layout issues iOS, iPhone

### ISSUE THREAD

**Issue by apofis1969** [airsdk/Adobe-Runtime-Support#458]
## Problem Description
Since adding the app to the iOS AppStore with 217 users have issues with the rendering of the layout. The app gets stuck.
It is a problem with creationcomplete at SkinnableContainer, HGroup, VGroup.
We can not reproduce this error. But we realised, that it effects device from USA and Canada, such as iPhone XS, 11, XS Max China and also iPhone X with iOS 13.5, 13.6, 13.7 
We tested also with our devices iPhone 8 and iPhone X, iPad Pro 12.9" 2nd generation and could not reproduce. Also tested with iOS 14 beta 8.
Basically this bug is only with the iPhone.
We removed now some of the creationcomplete and hope, that this error will not appear anymore. 
This is new in 33.1.1.217
If this error still exists we have to move back to 33.1.1.98 which is the best working SDK with iOS.
Also Bindable variables for paddingTop, paddingBotton,... create problems since 217.

**Comment by ajwfrost** [comment]
Hi - we've had some reports of this with 217 which is odd as there are no changes in there that should impact the Flex SDK behaviour. creationComplete event is sent out by the Flex framework, and isn't anything to do with AIR..

Do you have a simple test app that demonstrates the problem that we could then run on both 217 and 98 to see what's happening under the hood? Or is it as simple as creating an app with an HGroup and seeing whether creationComplete is received?

thanks

**Comment by apofis1969** [comment]
No, I do not have it. Tried to reproduce but no chance. It seems that the sdk does not take the layout settings after the launch screen and gets stuck. No idea why.

**Comment by ajwfrost** [comment]
Okay - we've had other reports of this on other platforms, one of the customers is trying to create a test app as they have it reproducible (I think on Windows); we also saw this happening on an Android app so may be able to debug that way..

**Comment by apofis1969** [comment]
Since removing the creation complete and Bindable variables the complaining stoped after initializing the app.
